packages/api/test/e2e/cennzx.e2e.ts

```diff
@@ -148,7 +149,7 @@ describe('CENNZX RPC calls testing', () => {
         // when I sell 1000(amount) PLUG, how much of CENTRAPAY will I get in return
         await expect(api.rpc.cennzx
           .sellPrice(PLUG, amount, CENTRAPAY)).rejects.toThrow(
-          'sellPrice(AssetToSell: AssetId, Amount: Balance, AssetToPayout: AssetId): Balance:: 2: Cannot exchange by requested amount.'
+          '2: Cannot exchange by requested amount.'
```

Comment:
Same here.

Reply:
This is how the RPC calls fails..
Although node has the error as - https://github.com/cennznet/cennznet/blob/b40b1afbd7a226823e08fb069b3b0b3ba92592aa/crml/cennzx/rpc/src/lib.rs#L145
But it throws with the index may be...